### ISSUE TITLE
Pools Average Hashrate in Pool Stats is confusing

### ISSUE THREAD

**Issue by pypper** [turtlecoin/block-explorer#37]
It's not super clear what the Average Hashrate and Average Difficulty are referring to in the Pools Stats block. 

Included screenshot for clarity.

![screenshot from 2018-03-30 16-02-37](https://user-images.githubusercontent.com/33644581/38152071-db73dd2c-3433-11e8-8e78-c9a4e74dcaad.png)

---

After reading the code on average hashrate, looks like it's calculating the average difficulty of recent blocks then figuring out the average hashrate over that time.

I'm guessing it was put in to show if the current hashrate and difficulty are higher or lower than they were recently, and there's probably a better way to present it.

given that we're calculating this average over block data that we have fetched -- there's no reason we can't graph network difficulty over the last X blocks.

Thoughts?

**Comment by yamiM0NSTER** [comment]
The problem is that Network speed is calculated based on results(dependent on luck). While Pool speed on first glance looks to be reported work of miners(actual hashes). We would have to either move to one or the other side to see real comparison.

**Comment by pypper** [comment]
Yes that is correct, how it works. Network speed is shown by calculating `lastStats.difficulty / blockTargetInterval`. While the average hashrate is calculating `(sum(recentDiffs) / recentDiffs.length) / blockTargetInterval`.

Without additional information in the UI, the average numbers can definitely be misleading.

I'm not sure these numbers were included in the first place to see a "real comparison", but to get a general feel for how the network is behaving. If this were the case, graphing these estimates against eachother would be less misleading than showing precise numbers in the stats section.

What do you propose as an alternative?
